examples/plugin/src/withShareExtensionInfoPlist.ts

```
@@ -0,0 +1,161 @@
+        UISceneConfigurations: {},
+      },
+      UIAppFonts: fonts.map((font) => path.basename(font)) ?? [],
+      // we need to add an AppGroup key for compatibility with react-native-mmkv https://github.com/mrousavy/react-native-mmkv
+      AppGroup: appGroup,
+      NSExtension: {
+        NSExtensionAttributes: {
+          NSExtensionActivationRule: activationRules.reduce((acc, current) => {
+            switch (current.type) {
+              case "image":
+                return {
+                  ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)